src/mongocxx/options/ssl.hpp

```diff
@@ -39,9 +39,12 @@ class MONGOCXX_API ssl {
     void ca_file(std::string ca_file);
     const bsoncxx::stdx::optional<std::string>& ca_file() const;
 
-    void ca_dir(std::string ca_file);
+    void ca_dir(std::string ca_dir);
```

Comment:
I wonder why we didn't ever comment these methods.  Since you are working in here anyway, can you add comments for these methods the way we've done it for other options?